fgprof/README.md

```diff
@@ -0,0 +1,59 @@
+---
+id: fgprof
+---
+
+# Fgprof
+
+![Release](https://img.shields.io/github/v/tag/gofiber/contrib?filter=fgprof*)
+[![Discord](https://img.shields.io/discord/704680098577514527?style=flat&label=%F0%9F%92%AC%20discord&color=00ACD7)](https://gofiber.io/discord)
+![Test](https://github.com/gofiber/contrib/workflows/Tests/badge.svg)
+![Security](https://github.com/gofiber/contrib/workflows/Security/badge.svg)
+![Linter](https://github.com/gofiber/contrib/workflows/Linter/badge.svg)
+
+[fgprof](https://github.com/felixge/fgprof) support for Fiber.
+
+**Note: Requires Go 1.19 and above**
+
+## Install
+
+This middleware supports Fiber v2.
+
+Using fgprof to profiling your Fiber app.
+
+```
+go get -u github.com/gofiber/fiber/v2
+go get -u github.com/gofiber/contrib/fgorof/v2
```

Comment:
Why is this `/v2` ?

Reply:
I see that other contribs use the v2 suffix to differentiate `fiber/v2`, which I misunderstood.

Reply:
That's because they are on v2.0.0, when a contrib is first created it's v1.0.0 until a breaking change is introduced